src/main/java/com/google/api/codegen/transformer/java/JavaSurfaceTransformer.java

```diff
@@ -122,9 +135,16 @@ public JavaSurfaceTransformer(
       InterfaceContext context =
           surfaceTransformer.createInterfaceContext(
               apiInterface, productConfig, namer, typeTable, enableStringFormatFunctions);
-      StaticLangFileView<StaticLangApiView> apiFile = generateApiFile(context, productConfig);
+
+      ApiAndSamples<StaticLangApiView, StaticLangApiMethodView> classViewWithSamples =
```

Comment:
If you have boolean words in your class name, you have the wrong abstraction.

Reply:
I'm happy to s/`And`/`With`/g, but that's not the point. This abstraction is there because samples are associated with methods. You don't have a sample floating off by itself: it always references a method. Contrariwise, a method will now come with one or more samples. That's the point of `ApiAndSamples`.

Reply:
ptal